### PR TITLE
Analytics 101

### PR DESCRIPTION
**Pull request by andresfulla**

### CODE REVIEW

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,2 +1,11 @@
+authored-by-x=Authored by {0}
+connect-to-analytics-cloud=Connect to Liferay Analytics Cloud
+connect-to-analytics-cloud-help=In order to view Content Performance, your Liferay DXP instance has to be connected with Liferay Analytics Cloud
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: Missing `.` at the end 🤷‍♀

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language_ar.properties [hunk 2]
```diff
@@ -1,2 +1,11 @@
+authored-by-x=Authored by {0} (Automatic Copy)
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language_ar.properties:1]: Build lang again...

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -1,2 +1,11 @@
...
 content-performance=Content Performance
-content-performance-panel=Content Performance
\ No newline at end of file
+content-performance-panel=Content Performance
+hide-content-performance-panel=Hide Content Performance Panel
+open-analytics-cloud=Open Analytics Cloud
+published-on-x=Published on {0}
+start-free-trial=Start Free Trial
+sync-to-analytics-cloud=Sync to Liferay Analytics Cloud
+sync-to-analytics-cloud-help=In order to view Content Performance, your site has to be synced with Liferay Analytics Cloud
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/content/Language.properties]: Missing `.` at the end 🤷‍♀

**Comment by andresfulla** [comment]
@dgarciasarai thansks!!! 🙌

Fixed those bits.

**Comment by andresfulla** [comment]
@darquesdev ready to go

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**